recipes/sundials/meta.yaml

```diff
@@ -26,6 +26,7 @@ requirements:
     - cmake
     - blas 1.1 {{ variant }}  # [not win]
     - openblas 0.2.18*  # [not win]
+    - python  # [win]
```

Comment:
Needs to be in `test` as well - the `run_test.bat` file is causing this, even if it's empty.